src/Tribe/Views/V2/Messages.php

```diff
@@ -84,6 +84,10 @@ public function __construct( $render_strategy = null, array $messages = [] ) {
 	public static function for_key( string $key, ...$values ) {
 		$map = [
 			'no_results_found'                 => __( 'There were no results found.', 'the-events-calendar' ),
+			'no_results_found_w_keyword'       => __(
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: b56ae0ce954e819df044d5200385be30</sub>